build/lib/pyaadhaar/tests.py

```diff
@@ -0,0 +1,24 @@
+from pdf2image import convert_from_path
+from pyzbar.pyzbar import decode
+from decode import AadhaarSecureQr
+
+filename = '/home/arkajit/Documents/Projects/pyaadhaar_updated/test_files/Test_Aadhar_Arkajit.pdf'
```

Comment:
Here it is the file path of file in your system
Either write a proper testcase which user can run or drop this file

Reply:
Sure will put some test cases, but I was worried about the information that'll go into the public domain.

Reply:
You can build a QR from the datastring provided in the Aaadhar Secure QR Docs as example